Apps/WebClient/src/ClientApp/src/views/faq.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+    <div class="m-3 px-4">
+        <b-row>
+            <b-col>
+                <b-row class="my-3 fluid">
+                    <b-col>
+                        <div id="pageTitle">
+                            <h1 id="subject">Frequently Asked Questions</h1>
+                            <hr />
+                        </div>
+                    </b-col>
+                </b-row>
+            </b-col>
+        </b-row>
+        <b-row>
+            <b-col>
+                <b-row>
+                    <b-col>
+                        <b-button
```

Comment:
Lets use cards and card headers
https://bootstrap-vue.org/docs/components/collapse#accordion-support